src/v3/widgetrc.ts

```diff
@@ -0,0 +1,96 @@
+import { WidgetOptions } from './src/types';
+
+const PROTOCOL = 'http';
+const HOSTNAME = 'localhost';
+const PORT = '3000';
+const ORIGIN = `${PROTOCOL}://${HOSTNAME}:${PORT}`;
+
+// common configs
+const common: Partial<WidgetOptions> = {
+  // target element
+  el: '#okta-signin-container',
+
+  // base url for building issuer
+  baseUrl: ORIGIN,
+
+  // https://github.com/okta/okta-signin-widget#logo
+  logo: '/img/widgico.png',
+
+  // https://github.com/okta/okta-signin-widget#logotext
+  logoText: 'widgico',
+
+  // https://github.com/okta/okta-signin-widget#language
+  language: 'en',
+
+  // https://github.com/okta/okta-signin-widget#redirecturi
+  redirectUri: `${ORIGIN}/login/callback`,
+
+  // https://github.com/okta/okta-signin-widget#feature-flags
+  features: {
```

Comment:
We might want to annotate this list with which flags are actually supported right now

Reply:
We should do this annotating before merge